app/src/main/java/inz/MainActivity.java

```diff
@@ -117,4 +135,24 @@ private void getFile(Uri pickerInitialUri) {
 
         startActivityForResult(intent, 100);
     }
-}
\ No newline at end of file
+
+    // creates menu
```

Comment:
It would be great to keep comment style unified through all of the files (in NeuralModel it was really good).